src/OpenTK/Math/Vector3.cs

```diff
@@ -933,6 +928,56 @@ public static Vector3 Transform(Vector3 vec, Matrix3 mat)
             return result;
         }
 
+        /// <summary>Transform a Vector by the given Matrix using right-handed notation</summary>
+        /// <param name="mat">The desired transformation</param>
+        /// <param name="vec">The vector to transform</param>
+        public static Vector3 Transform(Matrix3 mat, Vector3 vec)
+        {
+            Vector3 result;
+            Transform(ref vec, ref mat, out result);
+            return result;
+        }
+
+        /// <summary>Transform a Vector by the given Matrix</summary>
+        /// <param name="vec">The vector to transform</param>
+        /// <param name="mat">The desired transformation</param>
+        /// <returns>The transformed vector</returns>
+        public static Vector3 Transform(Vector3 vec, Matrix4 mat)
+        {
+            Vector3 result;
+            Transform(ref vec, ref mat, out result);
+            return result;
+        }
+
+        /// <summary>Transform a Vector by the given Matrix</summary>
+        /// <remarks>
+        /// It is incorrect to call this method passing the same variable for
```

Comment:
Maybe use this PR to fix the build warning:
`Math/Vector3d.cs(932,62): warning CS1734: XMLcomment on 'Vector3d.Transform(ref Vector3d, ref Matrix4d, out Vector3d)' has a paramref tag for 'left', but there is no parameter by that name
        Math/Vector3d.cs(933,29): warning CS1734: XMLcomment on 'Vector3d.Transform(ref Vector3d, ref Matrix4d, out Vector3d)' has a paramref tag for 'right', but there is no parameter by that name`

Reply:
Let's keep documentation changes separate from implementation changes.